lib/src/base-styles/_nx-lists.scss

```diff
@@ -20,101 +22,96 @@
   }
 }
 
-.nx-list__title {
-  color: $nx-text-color;
-  margin: 0;
-}
-
 .nx-list__item {
-  @include semi-bold();
-  @include nx-truncate-ellipsis();
+  @include container-horizontal;
   @include container-vertical;
 
-  border-top: 1px solid transparent;
-  border-bottom: $nx-border-row;
+  border-top: 1px solid #e9e9e9;
   box-sizing: border-box;
-  color: $nx-text-color;
+  color: $nx-text-color-light;
+  display: grid;
+  grid-template-columns: 85% auto;
+  grid-template-rows: auto auto;
   margin: 0;
-  padding: $nx-spacing-sm $nx-spacing-xs;
+  padding: $nx-spacing-md;
 
   // we use first of type because we only want to modify the first list item,
   // not alerts, errors, or empty states
   &:first-of-type {
-    border-top: $nx-border;
+    border-top: 1px solid #d1d6f0;
   }
+}
 
-  &.nx-error {
-    border: $nx-border-error;
-    color: $nx-text-color-error;
-  }
+.nx-list__text {
+  @include semi-bold();
 
-  > .fa, > .svg-inline--fa {
-    margin-right: $nx-spacing-md;
-  }
+  grid-column: 1;
+  grid-row: 1;
 }
 
-.nx-list--bulleted {
-  .nx-list__item {
-    @include regular();
+.nx-list__subtext {
+  @include regular();
 
-    border: none;
-    list-style: disc outside;
-    margin: $nx-spacing-sm 10px $nx-spacing-sm 36px;
-    overflow: visible;
-    padding: 0;
-    white-space: normal;
-  }
+  align-self: start;
+  display: block;
+  grid-column: 1;
+  grid-row: 2;
+  // top margin is measured down from the top of the objects grid row
+  margin: $nx-spacing-sm 0 0 0;
+  white-space: normal;
 
-  .nx-list__item {
-    list-style-type: disc;
+  &.nx-truncate-ellipsis {
+    white-space: nowrap;
+    width: $nx-paragraph-width-maximum;
+  }
+}
 
-    .nx-list__item {
-      list-style-type: circle;
+.nx-list__item--empty {
+  display: list-item;
+  text-align: center;
 
-      .nx-list__item {
-        list-style-type: square;
-      }
-    }
+  .nx-list__text {
+    @include regular();
   }
 }
 
-.nx-list__item--empty {
-  @include regular();
+.nx-list__item--error {
+  display: list-item;
+  padding-top: 0;
```

Comment:
It seems slightly odd that the error styles use margin to space the alert from the top border, but the empty styles use _padding_ to space the empty message from the top border.  It also seems odd that one uses 16px and the other uses 32px, but that's a question for @dturner32 .

Reply:
> It seems slightly odd that the error styles use margin to space the alert from the top border, but the empty styles use _padding_ to space the empty message from the top border. It also seems odd that one uses 16px and the other uses 32px, but that's a question for @dturner32 .

they should both have a padding of 16px

Reply:
> > It seems slightly odd that the error styles use margin to space the alert from the top border, but the empty styles use _padding_ to space the empty message from the top border. It also seems odd that one uses 16px and the other uses 32px, but that's a question for @dturner32 .
> 
> they should both have a padding of 16px

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/126907a37cf454a19189525fefa5959d4cdb1075